scripts/v.in.wfs/v.in.wfs.py

```diff
@@ -45,13 +46,27 @@
 # % required: no
 # %end
 # %option
+# % key: layer
+# % type: string
+# % description: name of data layers to import
+# % multiple: yes
+# % required: yes
+# %end
+# %option
 # % key: srs
 # % type: string
 # % label: Specify alternate spatial reference system (example: EPSG:4326)
 # % description: The given code must be supported by the server, consult the capabilities file
 # % required: no
 # %end
 # %option
+# % key: aoi
```

Comment:
Is it not enough to set the region with the combination with the `-r` flag (Restrict fetch to features which touch the current region) instead of this new `aoi` param?